runtime/bindings.m

```diff
@@ -98,25 +98,33 @@
 
 typedef CGPoint (*vision_func) (vector_float2 faceLandmarkPoint, CGRect faceBoundingBox, size_t imageWidth, size_t imageHeight);
 
+static void* vision_handle;
```

Comment:
Make this a static field? No use exporting it outside the function if nothing uses it.